src/Microsoft.VisualStudio.AppDesigner/PropPageDesigner/PropPageDesignerView.vb

```diff
@@ -536,6 +536,11 @@ Namespace Microsoft.VisualStudio.Editors.PropPageDesigner
             Select Case msg
                 Case Win32Constant.WM_PALETTECHANGED, Win32Constant.WM_SYSCOLORCHANGE, Win32Constant.WM_THEMECHANGED
                     OnThemeChanged()
+                Case Win32Constant.WM_SETTINGCHANGE
```

Comment:
This seems like a very broad message. Is there any way to narrow this down further so we only do work when the font changes, or is this the best we can do?